code-the-dream-react/src/App.js

```diff
@@ -1,22 +1,38 @@
 import * as React from 'react';
+import { useEffect } from 'react';
 import { TodoList } from './TodoList.js';
 import { AddTodoForm } from './AddTodoForm.js';
  
-function App() {
-  const [todoList, setTodoList] = React.useState([]);
+function useSemiPersistentState() {
+  const [todoList, setTodoList] = React.useState(() => {
```

Comment:
not a biggie, but this line could be simplified using just `useState` instead of `React.useState` just import it the same as you are importing `useEffect`. 
This will allow you to remove this line `import * as React from 'react';`

Reply:
Ah, ok. I was wondering that! Thank you! I've updated this upon submission of lesson 1.7

Reply:
[Resolved in Lesson 1.7](https://github.com/fernperezg97/react-todo/pull/7)